steam_wrapper/steam_wrapper.py

```diff
@@ -125,12 +125,14 @@ def read_file(path):
 def check_device_perms():
     has_perms = False
     logging.debug("Checking input devices permissions")
-    for entry in posix1e.ACL(file="/dev/uinput"):
-        if (entry.tag_type == posix1e.ACL_USER
-            and entry.qualifier == os.geteuid()
-            and entry.permset.write):
-            has_perms = True
-            break
+    uinput_path = Path("/dev/uinput")
+    if uinput_path.exists():
```

Comment:
I'd rather `return None` if the path doesn't exist. It doesn't make sense to show the "no input device permissions" message if the device is unavailable.